README.md

```diff
@@ -67,13 +67,13 @@ cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html),
 
 For `cudf version == 21.081.06` :
 ```bash
-# for CUDA 10.1
+# for CUDA 11.0
 conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=21.081.06 python=3.7 cudatoolkit=10.1
+    cudf=21.081.06 python=3.7 cudatoolkit=11.0
```

Comment:
the versions here look wrong

Reply:
Resolving it in a separate PR: https://github.com/rapidsai/cudf/pull/8331